www/src/components/PropsTable.tsx

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { MDXRenderer } from 'gatsby-plugin-mdx';
-import { Badge, Card } from '~paragon-react';
+import { Badge, Card, Hyperlink } from '~paragon-react';
 import PropType from './PropType';
 
 const IGNORED_COMPONENT_PROPS = ['intl'];
 
+const bootstrapLinks = {
+  Button: 'https://react-bootstrap-v4.netlify.app/components/buttons/#button-props',
+  Card: 'https://react-bootstrap-v4.netlify.app/components/cards/#card-props',
+  CardBody: 'https://react-bootstrap-v4.netlify.app/components/cards/#card-body-props',
+  CardDeck: 'https://react-bootstrap-v4.netlify.app/components/cards/#card-deck-props',
+  Dropdown: 'https://react-bootstrap-v4.netlify.app/components/dropdowns/#dropdown-props',
+  DropdownToggle: 'https://react-bootstrap-v4.netlify.app/components/dropdowns/#dropdown-toggle-props',
+  DropdownItem: 'https://react-bootstrap-v4.netlify.app/components/dropdowns/#dropdown-item-props',
+  DropdownMenu: 'https://react-bootstrap-v4.netlify.app/components/dropdowns/#dropdown-menu-props',
+  DropdownButton: 'https://react-bootstrap-v4.netlify.app/components/dropdowns/#dropdown-button-props',
+  FormControl: 'https://react-bootstrap-v4.netlify.app/components/forms/#form-control-props',
+  Nav: 'https://react-bootstrap-v4.netlify.app/components/navs/#nav-props',
+  WrapperPopover: 'https://react-bootstrap-v4.netlify.app/components/overlays/#popover-props',
```

Comment:
This one is really weird, `WrapperPopover` is the internal component name, but we export it and use under `Popover` name. In Props table it also says `WrapperPopover Props API` which is confusing, can you try fixing this so it says `Popover` instead of `WrapperPopover`?

Reply:
Good catch! Done